app/code/core/Mage/CatalogRule/Model/Rule/Action/Product.php

```diff
@@ -42,10 +42,10 @@ public function loadAttributeOptions()
     public function loadOperatorOptions()
     {
         $this->setOperatorOption([
-            'to_fixed' => Mage::helper('cataloginventory')->__('To Fixed Value'),
-            'to_percent' => Mage::helper('cataloginventory')->__('To Percentage'),
-            'by_fixed' => Mage::helper('cataloginventory')->__('By Fixed value'),
-            'by_percent' => Mage::helper('cataloginventory')->__('By Percentage'),
+            'to_fixed'   => static::$translate ? Mage::helper('cataloginventory')->__('To Fixed Value') : 'To Fixed Value',
+            'to_percent' => static::$translate ? Mage::helper('cataloginventory')->__('To Percentage') : 'To Percentage',
+            'by_fixed'   => static::$translate ? Mage::helper('cataloginventory')->__('By Fixed value') : 'By Fixed value',
+            'by_percent' => static::$translate ? Mage::helper('cataloginventory')->__('By Percentage') : 'By Percentage',
         ]);
```

Comment:
Not sure if this works:
```suggestion
        if (static::$isAdmin) {
            $this->setOperatorOption([
                'to_fixed' => Mage::helper('cataloginventory')->__('To Fixed Value'),
                'to_percent' => Mage::helper('cataloginventory')->__('To Percentage'),
                'by_fixed' => Mage::helper('cataloginventory')->__('By Fixed value'),
                'by_percent' => Mage::helper('cataloginventory')->__('By Percentage'),
            ]);
        } else {
            $this->setOperatorOption([]);
       }
```
`$isAdmin` is more meaningful as `$translate` is applicable in frontend.  It reduces the number of comparisons to just one.